### ISSUE TITLE
Server Error in Application /.

### ISSUE THREAD

**Issue by YounesOuteha** [fuel9/DisplayMonkey#199]
Hello,
I am hosted my app in my local IIS, I manage to connect to the DisplayMonkey. But when I try to connect to the DisplayMonkey_Management I got an error: "Server Error in Application /."
![Error](https://user-images.githubusercontent.com/56351753/116886625-96e29380-ac29-11eb-8aa1-8fd221ea0820.PNG)

**Comment by Plesmanweg** [comment]
sie mussen noch etwas umstellen nach off, gleich wie in die fehler steht. :)

**Comment by YounesOuteha** [comment]
Können Sie mir bitte helfen was genau soll ich tun, ich bin relativ neu.

**Comment by Plesmanweg** [comment]
sie mussen den web.config öffnen mitt notepad++ oder etwas.
Und dort sollte ein regel sein mit CustomErrormode und dort mussen sie von "on" > off machen.

**Comment by YounesOuteha** [comment]
Ja, ich habe schon aber trotzdem die Website ist nicht erreichbar.

**Comment by Plesmanweg** [comment]
oke, und die sql database und die login daten sind auch richtig?

**Comment by YounesOuteha** [comment]
ja, also die DisplayMonkey funktioniert schon, das Problem tritt nur bei der MonkeyDisplaya_Management App auf.

**Comment by Plesmanweg** [comment]
oke, brauchen sie auch den port 8080 hinter den ip oder dns name von de displaymonkey_management?

http://"ip_oder_dns_von_displaymonkey_server":8080/DisplayMonkey_Management

**Comment by YounesOuteha** [comment]
Ja, das habe ich auch.

**Comment by YounesOuteha** [comment]
unter port 82

**Comment by Plesmanweg** [comment]
oke, auch mal mitt 8080 geprobiert?

**Comment by YounesOuteha** [comment]
Ja, jetzt bekomme ich hier den Fehler.
![error1](https://user-images.githubusercontent.com/56351753/116988293-77eb0c80-acd0-11eb-8935-5363bac593ab.PNG)

**Comment by YounesOuteha** [comment]
Ich denke jetzt steigt der Fehler aufgrund des in der Datenbank gespeicherten Benutzers

**Comment by Plesmanweg** [comment]
sie mussen noch den benutzer in ein regel in den web.config setzen. 
der regel begint mitt das folgende:  <add name="DisplayMonkeyEntities" connectionString="metadata 

Und sie mussen ein lokales benutzer haben:  data source=localhost\DisplayMonkey

**Comment by YounesOuteha** [comment]
<add name="DisplayMonkeyEntities" connectionString="metadata=res://*/Models.DMModel.csdl|res://*/Models.DMModel.ssdl|res://*/Models.DMModel.msl;provider=System.Data.SqlClient;provider connection string=&quot;### **_data source=MGM\SQLEXPRESS;initial catalog=DisplayMonkey;persist security info=True;user id=DisplayMoneky;password=123456#_**;MultipleActiveResultSets=True;App=EntityFramework&quot;" providerName="System.Data.EntityClient" />

**Comment by YounesOuteha** [comment]
Das habe ich auch schon.

**Comment by YounesOuteha** [comment]
data source=MGM\SQLEXPRESS;initial catalog=DisplayMonkey;persist security info=True;user id=DisplayMoneky;password=123456#

**Comment by Plesmanweg** [comment]
Ich denke die data source ist nicht richtig?
oder heist deine sql datenbank fur displaymonkey:  SQLEXPRESS? und hat kein andere name?

**Comment by YounesOuteha** [comment]
Ja das heißt so.

**Comment by Plesmanweg** [comment]
dan ist nog was loss mitt den datenbank connection und den benutzer "displaymonkey"

**Comment by YounesOuteha** [comment]
Ja genau, aber ich kann das Problem nicht finden. Alles scheint richtig zu sein.

**Comment by YounesOuteha** [comment]
trotzdem Danke.

**Comment by Plesmanweg** [comment]
bitte sehr hoffe sie kommen noch daraus :)

**Comment by Plesmanweg** [comment]
when sie problemen hast mit powerbi oder etwas dan können sie auch noch mall kucken ob die dot einstellung in den web.config gut steht. 
Wen sie dotnetfx 4.7.1 drehen hast dan muss das auch noch angepast worden in den web.config in den displaymonkey_management Folder.  

```
<compilation targetFramework="4.7.1">
      <assemblies>
        <add assembly="System.Data.Entity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
      </assemblies>
    </compilation>
    <httpRuntime targetFramework="4.7" maxRequestLength="1048576" executionTimeout="120000" />
        <authentication mode="Windows" />

```

Und den web.config in den displaymonkey folder geändert worden.
```
 <system.web>
    <compilation targetFramework="4.7" />
	<httpRuntime targetFramework="4.7" />
    <authentication mode="Forms">
```

**Comment by YounesOuteha** [comment]
Ich habe verändert aber leider ist nicht geklappt.

**Comment by Plesmanweg** [comment]
Und wen sie noch localhost ändert in deine server name wo die displaymonkey sql datenbank aufsteht beim data source?
data source=localhost\DisplayMonkey

**Comment by YounesOuteha** [comment]
Das hat nicht geholfen, wir haben die Lösung aber gefunden. Es lag ganz simpel an einem Authorisierungsfehler. Nun haben wir aber festgestellt, dass Exchange nur bis Version 2013 SP1 unterstützt wird. Gibt es eine Lösung für 2016er Exchange?

**Comment by Plesmanweg** [comment]
eigentlich nicht, wir brauch das nicht.
Aber ich denke wen sie programmieren kannst das sie das in den source vielleicht anpassen kann?
Displaymonkey ist nämlich opensource.

Oder vielleicht Nur EWS von office/ microsoft 365 gebrauchen?

**Comment by YounesOuteha** [comment]
Eigentlich brauche ich den Exchange, aber ich werde vielleicht diese Version verwenden.

**Comment by Plesmanweg** [comment]
oke, sie haben kein exchange online?

**Comment by YounesOuteha** [comment]
was meinst du?

**Comment by Plesmanweg** [comment]
Microsoft hast auch ein online exchange version. 
Dan brauchst du kein on premise exchange server. :)

**Comment by YounesOuteha** [comment]
Okay, ich werde es überprüfen, hängt von der Notwendigkeit ab. Und Vielen Dank. :)

**Comment by YounesOuteha** [comment]
Hallo, ich habe es geschafft aber jetzt kann ich nicht den Profil erstellen, es bekomme immer ein Fehler das die Email die ich eingebe ist Falsche, hast du Idee dazu!? Danke

**Comment by Plesmanweg** [comment]
nein, hab dar kein Antwort auf. Ich brauche den Funktion nicht im office so.

**Comment by YounesOuteha** [comment]
wie kann man das machen? hast du einen Tipp? Wie benutze du die App?

**Comment by Plesmanweg** [comment]
wir brauchen displaymonkey nur fur showen von PowerBI Reports and some youtube video un websites und bilder.
Wir brauchen kein exchange.
Und das login am management pagina von Displaymonkey geht mit unsere server domain login.
